Packs/CheckpointFirewall/ReleaseNotes/2_0_9.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CheckPoint Firewall v2
+- Fixed an issue where the ***checkpoint-show-objects*** command failed.
```

Comment:
Please be more specific in the RN - mention that the error from the API was not handled correctly. Also mention the version API fix